schemas/mw-page-move.yaml

```diff
@@ -0,0 +1,66 @@
+title: MediaWiki Page Move
+description: Represents a MW Page Move event
+type: object
+properties:
+  # global event fields
+  event:
+    type: object
+    properties:
+      uri:
+        type: string
+        format: uri
+        description: the unique URI identifying the event
+      request_id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique UUID v1 ID of the event derived from the X-Request-Id header
+      id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique ID of this event; should match the ts field
+      ts:
+        type: string
+        format: date-time
+        description: 'the time stamp of the event, in ISO8601 format'
+      domain:
+        type: string
+        description: the domain the event pertains to
+    required:
+      - uri
+      - request_id
+      - id
+      - ts
+      - domain
+  # event-specific fields
+  new_title:
+    type: string
+    description: the new title of the page
+  old_title:
+    type: string
+    description: the old title of the page
+  page_id:
+    type: integer
+    minimum: 1
+    description: the page ID of the moved page
+  old_revision:
+    type: integer
+    minimum: 1
+    description: the last revision ID before the move
+  new_revision:
+    type: integer
+    minimum: 1
+    description: the first revision ID after the move
+  user_id:
+    type: string
+    description: the user that performed the move
+  user_text:
+    type: string
+    description: the text representation of the user
+  summary:
+    type: string
+    description: the summary comment left by the user
+required:
+  - event
+  - new_title
+  - old_title
+  - page_id
```

Comment:
It's possible to move the page across namespaces, so it could worth adding `old_namespace` and `new_namespace`